_posts/2019-01-08-Golang.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: 'Golang Interface for TVM Runtime'
+author: Siva
+date: 2019-01-08
+---
+## Author
+
+Contributed by Huawei Technologies India Pvt Ltd, Deep Learning Group.
```

Comment:
As per apache tradition, we prefer contributor as individuals, you can list your name as author and add an acknowledgement to the group in the end.

Reply:
Never mind :). I just borrowed template from other blogs.